images/ipfailover/keepalived/lib/config-generators.sh

```diff
@@ -250,8 +250,28 @@ $(generate_script_config "${ipaddr}" "${port}")
 
   local counter=1
   local previous="none"
+  local vip_counter=0
+  local total_vips=( $vips )
+  local vips_per_group=1
+  local vips_mod=0
+
+  if [[ $vip_groups -gt 0 ]]; then
+    vips_per_group=$((${#total_vips[@]} / vip_groups))
+    vips_mod=$((${#total_vips[@]} % vip_groups))
+  fi
+
+  while [[ "${vip_counter}" -lt "${#total_vips[@]}" ]]; do
+    local cur_vip_count=vips_per_group
+
+
```

Comment:
i don' think we need more than one blank line to be clear/easy to read.

Reply:
most of the file has consistent two space indentantion

Reply:
Yes two space but not two blank lines.